metrics/local_test.go

```diff
@@ -50,10 +51,11 @@ func TestLocalMetrics(t *testing.T) {
 	require.NotNil(t, g)
 
 	assert.Equal(t, map[string]int64{
-		"my-counter|x=y":           10,
-		"my-counter":               6,
-		"other-counter":            8,
-		"namespace.my-counter|x=y": 7,
+		"my-counter|x=y":             10,
+		"my-counter":                 6,
+		"other-counter":              8,
+		"namespace.my-counter|x=y":   7,
+		"ns.subns|service=a-service": 9,
```

Comment:
The current master would return this metric as `ns.subns.|service=a-service` (extra dot at the end, before the `|`)